tests/python/pytorch/graphbolt/gb_test_utils.py

```diff
@@ -250,6 +256,16 @@ def genereate_raw_data_for_hetero_dataset(
         np.save(os.path.join(test_dir, node_feat_path), node_feats)
         node_feats_path[ntype] = node_feat_path
 
+    # Generate edge features.
+    edge_feats_path = {}
+    os.makedirs(os.path.join(test_dir, "data"), exist_ok=True)
+    for etype, num_edge in num_edges.items():
+        src_ntype, etype_str, dst_ntype = etype
+        edge_feat_path = os.path.join("data", f"{etype_str}-feat.npy")
+        edge_feats = np.random.rand(num_edge, num_classes)
```

Comment:
Instead of writing a random value, is it possible to write a deterministic tensor so you can test the retrieved value?

Reply:
I'm inclined to keep it random here for consistency with the code above. We can change it to a deterministic tensor in another PR.